FluidNC/src/Maslow/Maslow.h

```diff
@@ -105,14 +105,12 @@ class Maslow_ {
     QWIICMUX I2CMux;
 
     //calibration stuff
-    float frame_width = 3500;
-    float frame_height = 2500;
 
     int frame_dimention_MIN = 1000;
     int frame_dimention_MAX = 5000;
     
     double calibrationGrid[CALIBRATION_GRID_SIZE][2] = {0};
-    float calibration_grid_offset = 750; // mm offset from the edge of the frame
+    float calibration_grid_offset = 620; // mm offset from the edge of the frame
```

Comment:
I think we should have it be centered instead of offset from the edge of the frame

Reply:
Not sure what you mean. It is centered, this just prevents the machine from running to close to the corners.